### PR TITLE
Add Add Photo Gallery tile for collective.cover

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

File: src/sc/photogallery/profiles/default/registry.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<registry xmlns:i18n="http://xml.zope.org/namespaces/i18n" i18n:domain="plone">
+  <record name="plone.app.tiles">
+    <field type="plone.registry.field.List">
+      <title>Tiles</title>
+      <value_type type="plone.registry.field.TextLine" />
+    </field>
+    <value purge="false">
```
Comment (rodfersou) [inline src/sc/photogallery/profiles/default/registry.xml]: I think this should be capitalized

Comment (hvelarde) [inline src/sc/photogallery/profiles/default/registry.xml]: lowercase is fine.

File: src/sc/photogallery/tiles/configure.zcml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    xmlns:plone="http://namespaces.plone.org/plone"
+    i18n_domain="brasil2016.portal">
+
+  <include package="plone.tiles" file="meta.zcml" />
+
+  <plone:tile
+      name="sc.photogallery"
+      title="Photo Gallery Tile"
+      description="A tile that shows a photogallery."
+      icon="++resource++brasil2016.portal/tile-photogallery.png"
```
Comment (rodfersou) [inline src/sc/photogallery/tiles/configure.zcml]: Should be sc.photogallery here

File: src/sc/photogallery/tiles/photogallery.pt [hunk 3]
```diff
@@ -0,0 +1,87 @@
+<html
+   xmlns="http://www.w3.org/1999/xhtml"
+   xmlns:metal="http://xml.zope.org/namespaces/metal"
+   xmlns:tal="http://xml.zope.org/namespaces/tal"
+   xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+   i18n:domain="brasil2016.portal">
```
Comment (rodfersou) [inline src/sc/photogallery/tiles/photogallery.pt]: Wrong domain

**Pull request merged**